app/controllers/organizations/staff/custom_pages_controller.rb

```diff
@@ -0,0 +1,25 @@
+class Organizations::Staff::CustomPagesController < Organizations::BaseController
```

Comment:
While you are back in this for the conflicts, can you please nest these modules/class instead of inline. We are trying to move to nested slowly but surely. Not a biggie if not, but sweet if you can.